src/agent/rustjail/src/cgroups/fs/mod.rs

```diff
@@ -37,6 +37,8 @@ use std::collections::HashMap;
 use std::fs;
 use std::path::Path;
 
+const GUEST_CPUSETS_PATH: &str = "/sys/devices/system/cpu/online";
```

Comment:
Maybe:

```suggestion
const GUEST_CPUS_PATH: &str = "/sys/devices/system/cpu/online";
```

Reply:
@egernst Yes, you are right, and updated.